db/migrate/20220712174111_add_posts_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :posts, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] To avoid having a reference column `posts_id`, kindly change `:posts` to `:post` on line `3` so that your foreign key has a singular name `post_id`. This is a good way of having your foreign key.